emacs.d/b-lsp.el

```diff
@@ -0,0 +1,19 @@
+(provide 'b-lsp)
+
+(require 'package)
+(setq package-user-dir (concat (file-name-directory load-file-name) "melpa"))
+(add-to-list 'package-archives '("melpa" . "https://melpa.org/packages/") t)
+(package-initialize)
+
+(require 'lsp-mode)
+(require 'lsp-pyright)
+(setq lsp-completion-provider :none)
+(setq lsp-pyright-auto-import-completions
+      nil)
+(setq lsp-signature-auto-activate nil) ;; disalbe bottom documentation popup
+(setq company-dabbrev-downcase 1)
```

Comment:
I had to look this up. Really odd name. Is company mode something we should use instead of dabbrev?

Reply:
Sorry, those were copy past bugs from my own config. 7a1ec86

I like the company-mode completions. It looks more "IDE like" in that you get a dropdown menu of completion and can cycle through them. But, not mandatory for lsp-mode.

Reply:
Thanks. I have seen that. It could be useful. Perhaps when there's time we'll look into it.